packages/sanity/src/core/store/_legacy/document/document-pair/operations/unpublish.ts

```diff
@@ -12,24 +10,25 @@ export const unpublish: OperationImpl<[], DisabledReason> = {
     }
     return snapshots.published ? false : 'NOT_PUBLISHED'
   },
-  execute: ({client, idPair, snapshots}) => {
-    let tx = client.observable.transaction().delete(idPair.publishedId)
-
-    if (snapshots.published) {
-      tx = tx.createIfNotExists({
-        ...omit(snapshots.published, '_updatedAt'),
-        _id: idPair.draftId,
-        _type: snapshots.published._type,
-      })
-    }
+  execute: ({client: globalClient, idPair}) => {
+    const vXClient = globalClient.withConfig({apiVersion: 'X'})
+    const {dataset} = globalClient.config()
 
-    return tx.commit({
+    return vXClient.observable.request({
+      url: `/data/actions/${dataset}`,
+      method: 'post',
+      //@todo awaiting backend support for this
+      // query: {skipCrossDatasetReferenceValidation: 'true'},
```

Comment:
I _think_ this should have landed by now(?)

Reply:
It is! Thank's for letting me know.